ui/packages/catalog-admin-module-sources/src/main/webapp/js/view/ModalSource.view.js

```diff
@@ -318,6 +318,9 @@ define([
                 view.checkName(view.$('.sourceName').find('input').val().trim());
                 view.renderDetails(config);
                 view.rebind(properties);
+                view.$el
+                .find('.control-group>.source-info')
+                .text(config.get('service').attributes.description)
```

Comment:
What happens here if the description is undefined?

Reply:
No error is thrown and the UI doesn't show anything where the description would go. We could protect against it, but that doesn't seem necessary.